sops/__init__.py

```diff
@@ -95,6 +95,8 @@
 
 INPUT_VERSION = VERSION
 
+DEFAULT_UNENCRYPTED_SUFFIX = '_unencrypted'
```

Comment:
I think based on how we handle `INPUT_VERSION` and our delay for #3, I will likely move to a global `UNENCRYPTED_SUFFIX` for developer sanity. I might keep a `DEFAULT_` still since it's nice to declare constants too.